infra/helm/meshforms/templates/secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: meshforms-secrets
+  namespace: {{ .Values.meshforms_app_namespace }}
```

Comment:
Looks like you are handing this via helm, may want to remove for consistency

Reply:
What do you mean by "this?" The secrets file? I use this within `deployment.yaml`, though it's not very templated and I need to fix that :/

Reply:
Namespace